### PR TITLE
Add ad layout shift docs

### PR DESCRIPTION
**Pull request by warrengm**
I messed up the commits -- good thing we squash them

### CODE REVIEW

File: docs/audits/cumulative-ad-shift.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Reduce ad layout shift
+
+## Overview
+
+Measures the total amount of user-visible layout shift that were caused by or
+happened near ads.
```
Comment (jimper) [inline docs/audits/cumulative-ad-shift.md:6]: Does this mean that the shift this metric measures can't always be directly attributed to ads? If so I think we should have some more details about the methodology used in the more info section.

Also suggest rewording to "...shift caused by or occurring near ads".

File: docs/audits/cumulative-ad-shift.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+This metric is not indicative of a specific issue. It should be used to help
+identify areas for improvement and track that improvement over time.
+
+To avoid layout shift, it's important to set sizes for ad elements before the
+ad tag loads. Ads are usually dynamically sized which can make it difficult or
+impossible to reserve the exact size of the ad ahead of time, but it's possible
+to reserve extra space to handle multi-size slots. More information about
+minimizing ad layout shift [can be found at the GPT reference]
+(https://developers.google.com/doubleclick-gpt/guides/minimize-layout-shift).
```
Comment (jimper) [inline docs/audits/cumulative-ad-shift.md:19]: This can be difficult or impossible to do perfectly—especially for ad slots that accept dynamically sized ads—but there are best practices you can follow to minimize layout shift in most case. For more information, see the guide to [minimizing layout shift](https://developers.google.com/doubleclick-gpt/guides/minimize-layout-shift) at the GPT developer site.

Comment (warrengm) [reply docs/audits/cumulative-ad-shift.md:19]: Thanks! Done

File: docs/audits/cumulative-ad-shift.md (hunk 1 above)
Comment (warrengm) [reply docs/audits/cumulative-ad-shift.md]: Went with  "...shift caused by or occurred near ads".

We can't distinguish cases where ads caused the shift or an ad shifted by some non-ad element.

Comment (jimper) [reply docs/audits/cumulative-ad-shift.md]: It should be "Measures the total amount of user-visible layout shift caused by or occurring near ads."

How would a user determine whether an ad is actually at fault or not, then?

Comment (warrengm) [reply docs/audits/cumulative-ad-shift.md]: Updated docs.

It's an opaque number. The plan is to follow up with more audits for attribution. But this metric lets publishers track the impact of changes locally and track the metric over time. Even for ad-caused layout shift, this number wouldn't show the fault (e.g. GPT, some first party script, the ad format, or some other ad script)

Comment (jimper) [reply docs/audits/cumulative-ad-shift.md]: That's fine, we can flesh the attribution part out later. We still need to be clear about what this metric is measuring in the meantime, though. I think a note in more info about how we're measuring this (layout instability API?) and known limitations of that approach would suffice.

Comment (warrengm) [reply docs/audits/cumulative-ad-shift.md]: PTAL

File: docs/audits/cumulative-ad-shift.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Reduce ad layout shift
```
Comment (jimper) [inline docs/audits/cumulative-ad-shift.md:1]: ad-related

File: docs/audits/cumulative-ad-shift.md (hunk 1 above)
Comment (jimper) [reply docs/audits/cumulative-ad-shift.md]: LGTM, thanks
